### PR TITLE
CRI-O: fix crio_image regexp

### PR DESCRIPTION
**Pull request by runcom**
Make it more robust cause it was failing with weird matches.

@stevekuznetsov @mrunalp PTAL

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml [hunk 1]
```diff
@@ -93,7 +93,7 @@ extensions:
       title: "origin prerequisites"
       repository: "aos-cd-jobs"
       script: |-
-        crio_tag="$( cat ./ORIGIN_PKG_VERSION | grep -Po '\d.\d' )"
+        crio_tag="$( cat ./ORIGIN_PKG_VERSION | grep -Po '\d\.\d' | head -1 )"
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: You don't need Perl regex for this

Comment (stevekuznetsov) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: And in what case does `ORIGIN_PKG_VERSION` have more than one line?

Comment (runcom) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: never, but if you get "-3.9.0-something3.4-something432" you'll get:
3.9
3.4

so, that's why the `head`

Comment (runcom) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: That's what's happening in https://github.com/openshift/openshift-ansible/pull/6449 for instance
```
-3.9.0-0.alpha.0.169.a2ec324

$ echo "-3.9.0-0.alpha.0.169.a2ec324" | grep -Po '\d\.\d'        1 ↵
3.9
0.1 
```

Comment (runcom) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: > You don't need Perl regex for this

do you have any suggestion about doing this w/o perl regexp? I'll happly use your command

Comment (stevekuznetsov) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: The current regex can be written without Perl syntax, but we can be more specific anyway. It is fair to assume that we could do:

```sh
crio_tag="$( grep -Po "(?<=^-)[0-9]+\.[0-9]+" ./ORIGIN_PKG_VERSION )"
```

Comment (runcom) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml]: fixed, thanks a ton @stevekuznetsov

**Comment by runcom** [comment]
Ping @stevekuznetsov :angel:

**Comment by stevekuznetsov** [comment]
```
$ sjb/push-update-automatic.sh 
[INFO] Checking for existence of test_branch_origin_extended_conformance_crio...
[INFO] Checking for existence of test_pull_request_openshift_ansible_extended_conformance_install_crio...
[INFO] Checking for existence of test_pull_request_origin_extended_conformance_crio...
[INFO] Updating test_branch_origin_extended_conformance_crio...
[INFO] Updating test_pull_request_openshift_ansible_extended_conformance_install_crio...
[INFO] Updating test_pull_request_origin_extended_conformance_crio...
```

**Pull request merged**